### PR TITLE
simplify installation page

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

Simplified the page, and put the gist of it at the top, this is the overview page, so it should just point to Cloud. Also removed some old stuff (added >2 yrs back).

the nightly vs stable section I don't like all that much, but I left it in.

### CODE REVIEW

File: packaging/installer/README.md [hunk 1]
```diff
@@ -1,44 +1,40 @@
...
+The easiest way to install Netdata on your system is via Netdata Cloud, to do so:
 
-Netdata runs permanently on all your physical/virtual servers, containers, cloud deployments, and edge/IoT devices.
-It runs on Linux distributions (Ubuntu, Debian, CentOS, and more), container/microservice platforms (Kubernetes clusters, Docker), and many other operating systems (FreeBSD, macOS), with no `sudo` required.
+1. Sign up to <https://app.netdata.cloud/>.
+2. You will be presented with an empty space, and a prompt to "Connect Nodes" with the install command for each platform.
+3. Select the platform you want to install Netdata to, copy and paste the script into your node's terminal, and run it.
 
-To install Netdata in minutes on your platform:
+Upon installation completing successfully, you should be able to see the node live in your Netdata Space and charts in the [Metrics tab](/docs/dashboards-and-charts/metrics-tab-and-single-node-tabs.md).
```
Comment (ilyam8) [inline packaging/installer/README.md:13]: Duplicate text

Screenshot 2024-05-27 at 14 45 23

Duplicate text

Screenshot 2024-05-27 at 14 45 23

Comment (Ancairon) [reply packaging/installer/README.md:13]: let me find the duplicate and see what to do about it

Comment (Ancairon) [reply packaging/installer/README.md:13]: wow, now I get it

Comment (Ancairon) [reply packaging/installer/README.md]: I must've pressed the revert button on something and added the previous wording back in

File: packaging/installer/README.md [hunk 2]
```diff
@@ -1,44 +1,34 @@
...
 
-To install Netdata in minutes on your platform:
+1. Sign up to <https://app.netdata.cloud/>.
+2. You will be presented with an empty space, and a prompt to "Connect Nodes" with the install command for each platform.
+3. Select the platform you want to install Netdata to, copy and paste the script into your node's terminal, and run it.
 
-1. Sign up to <https://app.netdata.cloud/>
-2. You will be presented with an empty space, and a prompt to "Connect Nodes" with the install command for each platform
-3. Select the platform you want to install Netdata to, copy and paste the script into your node's terminal, and run it
+Upon installation completing successfully, you should be able to see the node live in your Netdata Space and charts in the [Metrics tab](/docs/dashboards-and-charts/metrics-tab-and-single-node-tabs.md).
```
Comment (ilyam8) [inline packaging/installer/README.md:13]:
```suggestion
Once Netdata is installed, you can see the node live in your Netdata Space and charts in the [Metrics tab](/docs/dashboards-and-charts/metrics-tab-and-single-node-tabs.md).
```

**Review by ilyam8 — APPROVED** [review]

File: packaging/installer/README.md (hunk 1 above)
Comment (ilyam8) [reply packaging/installer/README.md]: > let me find the duplicate and see what to do about it

I even posted duplicate report of this text!

**Pull request merged**